dev/core/src/com/google/gwt/dev/js/JsSafeCloner.java

```diff
@@ -143,13 +143,17 @@ public void endVisit(JsNameOf x, JsContext ctx) {
      */
     @Override
     public void endVisit(JsNameRef x, JsContext ctx) {
-      if (x.getQualifier() == null && x.getIdent() == "arguments") {
+      if (x.getQualifier() == null && "arguments".equals(x.getIdent())) {
```

Comment:
Same here Objects.isNull

Reply:
Objects.equals() probably? I think that would be better than what I have here now, as a replacement for `==`. I've never seen this fail, but it certainly could, and then would break here, which is why I changed it.